cmd/test/test.go

```diff
@@ -0,0 +1,228 @@
+// Copyright The Enterprise Contract Contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+// -------------------------------------------------------------------------------
+// This file is intentionally almost to identical to the conftest version of this
+// command. Use `make conftest-test-cmd-diff` to show a comparison.
+// -------------------------------------------------------------------------------
+package test
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/open-policy-agent/conftest/output"
+	"github.com/open-policy-agent/conftest/parser"
+	"github.com/open-policy-agent/conftest/runner"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	"github.com/enterprise-contract/ec-cli/internal/applicationsnapshot"
+)
+
+const (
+	OutputAppstudio = "appstudio"
+)
```

Comment:
Parentheses are not needed when defining a single constant as you're doing here. Either remove parentheses or include line numbers 41-97 within parenthetical enclosures.

Reply:
Fixed in latest revision.